buildsupport/groovy/pom.xml

```diff
@@ -16,6 +16,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
   <modelVersion>4.0.0</modelVersion>
+  <properties>
+    <groovy.version>2.3.7</groovy.version>
```

Comment:
should be able to got to 2.3.9, but they do recommend using matching versions of eclipse-batch